ipatests/test_integration/test_caless.py

```diff
@@ -121,7 +121,9 @@ class CALessBase(IntegrationTest):
     @classmethod
     def install(cls, mh):
         cls.cert_dir = tempfile.mkdtemp(prefix="ipatest-")
-        cls.pem_filename = os.path.join(cls.cert_dir, 'root.pem')
+        cls.pem_filename = 'root.pem'
+        cls.ca2_crt = 'ca2_crt.pem'
+        cls.ca2_kdc_crt = 'ca2_kdc_crt.pem'
```

Comment:
As far as I can see, you are using ``pem_filename`` and the other file names with ``os.path.join(cert_dir, ...)`` all the time. Rather than calling join all the time, you should make ``cls.pem_filename`` an absolute path.